lib/EventDispatcher/DebugEventDispatcher.php

```diff
@@ -23,6 +23,11 @@
 class DebugEventDispatcher extends ContainerAwareEventDispatcher
 {
     /**
+     * @var int
+     */
+    private $depth = 0;
```

Comment:
If there is an exception then the program is probably going to exit. This is just a debugging tool ...